Source/WebInspectorUI/UserInterface/Views/DOMTreeContentView.js

```diff
@@ -694,6 +702,77 @@ WI.DOMTreeContentView = class DOMTreeContentView extends WI.ContentView
         this._showPrintStylesChanged();
     }
 
+    _handleAccessibilitySettingsButtonClicked()
+    {
+        if (!this._toggleAccessibilitySettingsNavigationItem.enabled)
+            return;
+
+        if (this._accessibilitySettingsPopover) {
+            this._accessibilitySettingsPopover.dismiss();
+            this._accessibilitySettingsPopover = null;
+            return;
+        }
+
+        const createCheckbox = (container, label, setting, value) => {
+            if (!setting)
+                return;
+
+            let labelElement = container.appendChild(document.createElement("label"));
+
+            let checkboxElement = labelElement.appendChild(document.createElement("input"));
+            checkboxElement.type = "checkbox";
+            checkboxElement.checked = value;
+            checkboxElement.addEventListener("change", (event) => {
+                this._setAccessibilitySettingForAllTargets(setting, event.target.checked);
+            });
+
+            labelElement.append(label);
+
+            return checkboxElement;
+        }
+
+        const calculateTargetFrame = () => {
+            return WI.Rect.rectFromClientRect(this._toggleAccessibilitySettingsNavigationItem.element.getBoundingClientRect()).pad(2);
+        }
+
+        const preferredEdges = [WI.RectEdge.MAX_Y, WI.RectEdge.MAX_X];
+
+        this._accessibilitySettingsPopover = new WI.Popover(this);
+        this._accessibilitySettingsPopover.windowResizeHandler = function(event) {
+            this._accessibilitySettingsPopover.present(calculateTargetFrame(), preferredEdges);
+        };
+
+        let contentElement = document.createElement("div");
+        contentElement.classList.add("accessibility-settings-content");
+
+        // COMPATIBILITY (macOS 13.3, iOS 16.4): `ForcedPrefersReducedMotion` value for `Page.Setting` did not exist yet.
+        if (InspectorBackend.Enum.Page?.Setting?.ForcedPrefersReducedMotion)
+            this._prefersReducedMotionCheckbox = createCheckbox(contentElement, "Reduce motion", InspectorBackend.Enum.Page.Setting.ForcedPrefersReducedMotion, WI.cssManager.prefersReducedMotion.enabled);
+
+        // COMPATIBILITY (macOS 13.3, iOS 16.4): `ForcedPrefersContrast` value for `Page.Setting` did not exist yet.
+        if (InspectorBackend.Enum.Page?.Setting?.ForcedPrefersContrast)
+            this._prefersContrastCheckbox = createCheckbox(contentElement, "Increase contrast", InspectorBackend.Enum.Page.Setting.ForcedPrefersContrast, WI.cssManager.prefersContrast.enabled);
+
+        this._accessibilitySettingsPopover.presentNewContentWithFrame(contentElement, calculateTargetFrame(), preferredEdges);
+    }
+
+    _setAccessibilitySettingForAllTargets(setting, value)
```

Comment:
please move this to `WI.CSSManager` as we prefer to keep protocol code out of view stuff